scripts/sql/create.sql

```diff
@@ -0,0 +1,41 @@
+-- Delegatees are account that receive delegation of voting power from Delegators
+CREATE TABLE Delegation
+(
+    token_address     CHAR(20),
+    delegator_address CHAR(20),
+    delegatee_address CHAR(20),
+    proof             VARCHAR,
```

Comment:
@joigno question: what is the maximum size for the varchar? 

_This may be better as a text field_

Reply:
This is probably betters as a `TEXT` field then 🤔

Reply:
`TEXT` is fine for me.